commons-lang3/pom.xml

```diff
@@ -140,7 +140,7 @@
 										<version>${core.artifact.version}</version>
 									</artifact>
 									<moduleInfoSource>
-										open module org.apache.commons.lang3 {
+										open module dev.ikm.jpms.commons.lang3 {
```

Comment:
Again, shouldn’t the module name remain `org.apache.commons.lang3`? Although it’s not mandatory, it would be consistent with the naming structure of the other modules in the JPMS repository, which generally matches the automatic module naming convention. A very good example is [Eclipse Collections](https://github.com/ikmdev/jpms-deps/blob/5538fefda73d454bdb0570714f1c0a8c14930581/eclipse-collections/pom.xml#L232)

Reply:
yeah, you found another bug in our releases.  Someone didn't understand how to publish eclipse collections.  That is a defect we will correct.